### ISSUE TITLE
Allow servers from a secure network to push their data through the Service broker (using certificate authentication)

### ISSUE THREAD

**Issue by NeoPangea** [sqlcollaborative/dbareports#86]
**Do you want to request a *feature* or report a *bug*?**
feature

**What is the current behavior?**
The dbareports server must have SQL level and WMI level access to the SQL instance being monitored

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via https://jsfiddle.net
If the dbareports server cannot connect to the server, it cannot be added to the inventory

**What is the expected behavior?**
I have a closed network containing card holder data. I need to be able to monitor the SQL Servers in this network but they are on their own domain with a trust, and protected via a firewall. I am allowed to make an outgoing controlled connection from inside this network to my monitoring server outside, but cannot make the connection the other way due to company policy and PCI requirements. I would like to create a job on my "island" server to push the data via service broker to the monitor server. The monitor server then needs to add the sender to the inventory and store it's data into the repository.

**Which versions of SQL Server and which OS are affected by this issue? Did this work in previous versions of our procedures?**
All supported versions

**Comment by wsmelton** [comment]
So basically: a job in dbareports that could look at a folder location to import data you send to the monitor server and import into the dbareports database?

**Comment by NeoPangea** [comment]
Yes that could work perfectly. The important part is that data cannot be pulled from the server. The server has to push it's data to the dbareports database (either via Service Broker, or even simply emailing the reporting data)
